bom/EAP7-1852_EAP_Jakarta_EE8_BOMs_name_and_content_changes.adoc

```diff
@@ -0,0 +1,84 @@
+= EAP Jakarta EE8 BOMs Name and Content Changes
+:author:            Eduardo Martins
+:email:             emartins@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+The Jakarta EE8 BOM (including its Maven artifactId jboss-eap-jakartaee8) and children BOMs need to be renamed, since EAP 8 will introduce Jakarta EE 10. For the same reason, the content of such BOMs, i.e the managed Maven dependencies related with the Jakarta EE platform, may also change.
+Besides the changes in the builders of such BOMs, we also need to update EAP documentation and application migration tooling, to help customers dealing with the changes of this proposal.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/EAP7-1852[EAP7-1852]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* Jan Stourac
+
+=== Testing By
+
+[ ] Engineering
+
+[x] QE
+
+=== Affected Projects or Components
+
+* WildFly, JBoss EAP
+
+=== Other Interested Projects
+
+* Migration Toolkit for Applications
+
+== Requirements
+
+=== Hard Requirements
+
+* BOMs Name Changes
+** The JBoss EAP `Jakarta EE 8` BOM should be renamed to JBoss EAP `EE` BOM, and its Maven artifactId coordinate `jboss-eap-jakartaee8` should be renamed to `jboss-eap-ee`. Similarly, the  Wildfly `Jakarta EE 8` BOM should be renamed to WildFly `EE` BOM, and its Maven artifactId coordinate `wildfly-jakartaee8` should be renamed to `wildfly-ee`.
+** The JBoss EAP `Jakarta EE 8 with Tools` BOM should be renamed to JBoss EAP `EE With Tools` BOM, and its Maven artifactId coordinate `jboss-eap-jakartaee8-with-tools` should be renamed to `jboss-eap-ee-with-tools`.  Similarly, the  Wildfly `Jakarta EE 8 With Tools` BOM should be renamed to WildFly `EE With Tools` BOM, and its Maven artifactId coordinate `wildfly-jakartaee8-with-tools` should be renamed to `wildfly-ee-with-tools`.
+
+* BOMs Content Changes
+** The Maven artifacts related with Jakarta EE 8 platform, which are managed dependencies in the BOMs referenced by this proposal, should be replaced by their successors in the Jakarta EE 10 platform. The same applies to any JBoss EAP component related with such artifacts that changes Maven Coordinates other than `version`.
+
+* BOMs Documentation
+** A new `EAP Jakarta EE8 BOMs Name and Content Changes` section should be added to the `Application Migration` chapter of the `EAP Migration Guide`, guiding customers to manually do the needed changes in their Maven Projects:
+*** Change the Maven Coordinates of the BOMs in Maven dependency management imports.
+*** Change the Maven Coordinates of the Jakarta EE platform artifacts in Maven dependencies.
+** Any other EAP Documentation reference to the `Jakarta EE 8` and `Jakarta EE 8 with Tools` BOMs should be updated to reflect the new names and Maven coordinates.
```

Comment:
To detailt better this bullet point we will need to track the new Development Guide in the works, and look for any outdated references to these BOMs.